saleor/discount/utils.py

```diff
@@ -55,11 +55,14 @@
 
 
 def increase_voucher_usage(
-    voucher: "Voucher", code: "VoucherCode", customer_email: str
+    voucher: "Voucher",
+    code: "VoucherCode",
+    customer_email: str,
+    allow_customer_voucher_creation: bool = True,
```

Comment:
I would suggest changing the name to `increase_voucher_customer_usage` or smth like this, as we're creating the `VoucherCustomer` under the hood but it's not visible in this function

Reply:
done